contracts/libraries/LibInventory.sol

```diff
@@ -6,16 +6,15 @@ LibInventory defines the storage structure used by the Inventory contract as a f
 proxy.
  */
 library LibInventory {
-  bytes32 constant STORAGE_POSITION = keccak256("summon.eth.storage.Inventory");
+  bytes32 constant STORAGE_POSITION = keccak256("eth.storage.Inventory");
```

Comment:
nit: minor but maybe we would call this lootledger.storage.Inventory vs eth as a prefix, just incase anyone is already using that position.

Reply:
yeah well, probably will be something different that "lootledger" (spoiler alert)